docs/conf.py

```diff
@@ -83,6 +85,11 @@
 .. |components_type| replace:: Union[:class:`disnake.ui.ActionRow`, :class:`disnake.ui.WrappedComponent`, List[Union[:class:`disnake.ui.ActionRow`, :class:`disnake.ui.WrappedComponent`, List[:class:`disnake.ui.WrappedComponent`]]]]
 .. |resource_type| replace:: Union[:class:`bytes`, :class:`.Asset`, :class:`.Emoji`, :class:`.PartialEmoji`, :class:`.StickerItem`, :class:`.Sticker`]
 .. _coroutine_link: https://docs.python.org/3/library/asyncio-task.html#coroutine
+.. |data_class| replace:: This class is a :ref:`Data Class <data_class>`.
+.. |data_class_constructible| replace:: This class is a constructible :ref:`Data Class <data_class>`.
+.. |discord_abc| replace:: This class is an :ref:`ABC <disnake_abc>`.
+.. |discord_model| replace:: This class is a :ref:`Discord Model <discord_model>`.
+.. |discord_enum| replace:: This class is an :ref:`Enumeration <discord_enum>`.
```

Comment:
I'm still undecided about having these at all, right now it just results in some duplication:
![image](https://user-images.githubusercontent.com/8530778/197403406-b159a197-bd27-48e2-b4da-b1eb66cf2178.png)

Reply:
removed in https://github.com/DisnakeDev/disnake/pull/722/commits/b18f2b84748354d9881e23866a841b683f535a89